docs: add section about getting logs not available on stdout (#1595)
SumoLogic/sumologic-kubernetes-collection: deploy/docs/Best_Practices.md

**NOTE** We observed that changing this value increases metrics loss during prometheus resharding,
but the traffic is much better balanced between Fluentds and Prometheus is more stable in terms of memory.

## Get logs not available on stdout

> When logs from Pod are not available on stdout [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
> helps with collecting them using standard logging pipeline.

Reviewer: ```suggestion
When logs from a Pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
can help with collecting them using standard logging pipeline.
```

Author: Corrected

> To tail logs using Tailing Sidecar Operator the file with logs needs to be accessible through volume
> because the volume provides the access to the log file and this volume is mounted to sidecar container.

Reviewer: ```suggestion
To tail logs using Tailing Sidecar Operator the file with those logs needs to be accessible through a volume
mounted to sidecar container.
```

Author: Corrected

  prometheus:
    prometheusSpec:
      scrapeInterval: '1m'
```

## Get logs not available on stdout

> When logs from a Pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)

Reviewer 2: ```suggestion
When logs from a pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
```
?

Author: I don't know how it should be written, In k8s docs Pod starts with capital letter, in other places it is written either as "Pod" or "pod"

Reviewer 2: ```
➜  sumologic-kubernetes-collection git:(drosiek-otc) ✗ grep ' pod ' **/**.md | wc -l
55
➜  sumologic-kubernetes-collection git:(drosiek-otc) ✗ grep ' Pod ' **/**.md | wc -l
7
```

I have no idea neither, by statistically we want to use `pod` :sweat_smile:
